interp/dumpglob.ml

```diff
@@ -51,13 +48,15 @@ let start_dump_glob ~vfile ~vofile =
       ()
 
 let end_dump_glob () =
-  match !glob_output with
+  match List.hd !glob_output with
   | MultFiles | File _ -> close_glob_file ()
   | NoGlob | Feedback -> ()
 
-let previous_state = ref MultFiles
-let pause () = previous_state := !glob_output; glob_output := NoGlob
-let continue () = glob_output := !previous_state
+let push_output d = glob_output := d::!glob_output
+let pop_output () = glob_output := match !glob_output with
+  | [] -> assert false
+  | [_] -> [NoGlob]
```

Comment:
That's weird, if you are gonna do this then use the empty list to signal no output. In fact, the code would look much better if you use an abstract data structure instead of the hardcoded `List.*` stuff.

Reply:
I have made the datastructure a bit more abstract (I assume this is what you meant?)

Reply:
Thanks! It is good for now.